### PR TITLE
refactor: move all parsers from core to new package

### PR DESCRIPTION
**Pull request by barak007**
This is the new place to experiment. Also this is for now a private package.

### CODE REVIEW

File: packages/experiments/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "name": "@tokey/experiments",
+  "description": "experimental tokenizers and parsers",
+  "version": "0.0.1",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "test": "mocha \"./dist/test/**/*.spec.js\""
+  },
+  "dependencies": {
+    "@tokey/core": "^1.1.0"
```
Comment (idoros) [inline packages/experiments/package.json:12]: 1.2.0

File: packages/experiments/test/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "../dist/test",
+    "types": ["mocha"]
+  },
+  "references": [{ "path": "../src" }]
```
Comment (idoros) [inline packages/experiments/test/tsconfig.json:7]: Is @tokey/test-kit missing?

Comment (barak007) [reply packages/experiments/test/tsconfig.json:7]: I works without it....not sure that it is needed

Comment (barak007) [reply packages/experiments/test/tsconfig.json:7]: yes

**Review by idoros — APPROVED** [review]

**Pull request merged**